pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.26.7
+version: 1.26.8-dev
```

Comment:
Generally I think it's a good idea to immediately release small fixes like this, unless we have immediate plans to make another change in the next day or two.

Reply:
Done